### ISSUE TITLE
[Eclipse][macOS] Cannot log in to CodeTogether account

### ISSUE THREAD

**Issue by seandevlin1** [Genuitec/CodeTogether#238]
CodeTogether's login functionality for **Eclipse macOS** client doesn't seem to be working.

I've verified that my CodeTogether account is active. Have also changed password several times and made login attempts with each new password, ensuring that login credentials are entered accurately in the Eclipse macOS client each time.

**CodeTogether Version: 3.1.0

Eclipse Version: 2020-12 (4.18.0)
Eclipse Build ID: 20201210-1552

macOS Version: High Sierra 10.13.6 (17G14042)**

![codetogether-eclipse-macos-login_issue](https://user-images.githubusercontent.com/38743415/104230842-1bf83180-541c-11eb-97a3-6841eeb26d07.png)

**Comment by brianvfernandes** [comment]
Sean, we checked and login services are fine and there were no recent outages / issues either. Can you try [resetting](https://www.codetogether.com/my-account/lost-password/) your password? I would recommend using a simple and short password for now, ensure you can log in, and then changing to the password of your choice.

**Comment by seandevlin1** [comment]
> Sean, we checked and login services are fine and there were no recent outages / issues either. Can you try [resetting](https://www.codetogether.com/my-account/lost-password/) your password? I would recommend using a simple and short password for now, ensure you can log in, and then changing to the password of your choice.

Thank you and the team for checking so quickly, Brian!

I took your advice and reset my password to a simple alphanumeric, 16-character one. This password allowed me to log in via Eclipse. I changed it back to a more complex password (40 character, letters, numbers, symbols) and login doesn't work again. So it seems that there's an issue with the password complexity I'm accustomed to using.

Are there password guidelines for CodeTogether accounts? Any character restrictions or length limitations?
